### PR TITLE
Raul Pedro redesign reports page

### PR DESCRIPTION
**Pull request by pedroelton**

### CODE REVIEW

File: src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 import { useSelector } from 'react-redux';
 import { PieChart } from 'components/common/PieChart';
-import { peopleTasksPieChartViewData } from '../selectors';
 import { ReportPage } from 'components/Reports/sharedComponents/ReportPage';
 import { NewModal } from 'components/common/NewModal';
+import { peopleTasksPieChartViewData } from '../selectors';
 import './PeopleTasksPieChart.css';
 
-export const PeopleTasksPieChart = () => {
+export function PeopleTasksPieChart() {
```
Comment (LeeSingChanYau) [inline src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx:14]: I am sure it works fine but why did you change from arrow function to normal?

**Review by LeeSingChanYau — COMMENTED** [review]
Good job Pedro! Changes look good! I ran your PR locally and I like how the new Reports Pages looks. The reports page new images look good. I also saw the changes in Team Reports. The aesthetic of TeamReports looks good, I like the addition of Pie Charts. But I think the pie charts are not displaying correctly all the information. I'll attach an image

File: src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx (hunk 1 above)
Comment (LeeSingChanYau) [reply src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx:14]: image

Comment (pedroelton) [reply src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx:14]: Hi Lee, I think it was those auto-fix. because I usually use the arrow function myself, I don't think I changed it intentionally.

**Comment by AnishPandita** [comment]
Hey @pedroelton I reviewed the new reports page implementation and it looks nice. The design and styling feels simple and elegant. As suggested by @LeeSingChanYau  the pie charts do however show wrong data.
image
image

**Comment by AnishPandita** [comment]
The code looks pretty clean too. Nothing to comment there.

File: src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx (hunk 1 above)
Comment (pedroelton) [reply src/components/Reports/PeopleReport/components/PeopleTasksPieChart.jsx:14]: This chart's values are wrong because they are hard coded. I didn't connect the data with the pie chart. I talked with Jae and a full-stack would take care of it because it has to do with the data from the database as well and I still can't do the backend.

**Comment by pedroelton** [comment]
> Hey @pedroelton I reviewed the new reports page implementation and it looks nice. The design and styling feels simple and elegant. As suggested by @LeeSingChanYau the pie charts do however show wrong data. image image

This chart's values are wrong because they are hard coded. I didn't connect the data with the pie chart. I talked with Jae and a full-stack would take care of it because it has to do with the data from the database as well and I still can't do the backend.

**Comment by pedroelton** [comment]
> Hey @pedroelton I reviewed the new reports page implementation and it looks nice. The design and styling feels simple and elegant. As suggested by @LeeSingChanYau the pie charts do however show wrong data. image image



This chart's values are wrong because they are hard coded. I didn't connect the data with the pie chart. I talked with Jae and a full-stack would take care of it because it has to do with the data from the database as well and I still can't do the backend.

**Comment by Gukal7** [comment]
@pedroelton The new design looks really intuitive, it's very easy to the eye and the animations are really nice too.
Screenshot 2023-01-27 at 5 09 04 PM
Screenshot 2023-01-27 at 5 11 08 PM

**Comment by Filipe-Oliv** [comment]
Hi @pedroelton 

The new page layout looks great, but I noticed some issues with different resolutions.

The cards on the Reports page are shifting position at a specific page size. Also, on the Team page where there are some pie chart, the whole page is not flexible.

[week 03 - 02-07-23 (9).webm](https://user-images.githubusercontent.com/114119668/217385004-d0a7d4cc-0483-47f0-bb0d-1d731dcc3fa8.webm)

**Comment by tingedudu8** [comment]
Hi @pedroelton, this new reports page looks nice but I have some suggestions.
1. The background of the first row of "Teams" is different from the other two. It is better to unify the style.
2. Since the number of people grows and it would be much better to split the display column into several pages.
3. You can see from the video, when the resolution changes, the display column will be affected.
Thanks!!

https://user-images.githubusercontent.com/70128933/217670518-9e480114-6657-4103-83e7-1c107fe898d6.mov

**Comment by PrabhjotSembhi** [comment]
@pedroelton hi, what's the status of this? Have your resolved all comments? and please resolve the conflict

**Review by suigunfjc — COMMENTED** [review]
@pedroelton Really love your design, they are very user-friendly and easy to use. Fantastic job!! 
Just wonder why the header row of the teams table is not in purple as the other header rows of projects and people. 
Screenshot 2023-02-18 180126

Also, it would be great if we can have a back button in detailed projectsreport, peoplereport and teamsreport pages.
Screenshot 2023-02-18 180437

**Comment by NyahJ** [comment]
Hi @pedroelton,

截屏2023-02-18 下午10 08 57
截屏2023-02-18 下午10 08 29

This is really awesome, the design and the function.  
A small suggestion: is it possible to make the part under "Select a Category" fixed? 
When we clicked "People" and "Team", it worked well and consistent. However, when the "Projects" is clicked (shown in the red circle),  all three parts are squeezed together, and the gaps in between disappears. Thanks!

**Comment by vitorsaa2k** [comment]
Really good PR @pedroelton! Loved the design.

I ran into a couple of issues on this PR:

1. The profile page seems broken

image
I dont now if this is related to this PR, correct me if Im wrong

2. There's no responseviness on the team's page
image

3. The dropdown is broken on the weekly summaries reports page, and also recommend remove this "page" in the end of the title of the page
image
I also dont know if this this is related to this PR

File: src/components/Reports/TeamReport/components/TestComponents/LoginPrivilegesSimulation.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+function LoginPrivilegesSimulation({ selectedInput, handleInputChange }) {
+//   // Create a state variable to store the selected radio input
+//   const [selectedInput, setSelectedInput] = useState('isManager');
+
+  //   // Event handler for when a radio input is selected
+  //   const handleInputChange = (event) => {
+  //     // Update the selectedInput state variable with the value of the selected radio input
+  //     setSelectedInput(event.target.value);
+  //   };
```
Comment (kamtoeddy) [inline src/components/Reports/TeamReport/components/TestComponents/LoginPrivilegesSimulation.jsx:15]: It would be nice to remove unused code rather than commenting

**Review by kamtoeddy — COMMENTED** [review]
Hi @pedroelton,

Tested your PR and the new UI is awesome.
All I have to say is that 
 - the code could be a little cleaner if you got rid of the commented code
 - you should look into the responsiveness as the other teammates have pointed out

Thanks

**Comment by dkamghem** [comment]
It seems to work. The project makes the app freeze.
Screen Shot 2023-03-02 at 9 05 28 PM (2)

**Comment by mpavithr** [comment]
Hi other than the pie chart and the inconsistent spacing between categories (see below)
2x2x
a6.PNG">
I was wondering whether the checkboxes seen below should be making changes to the pie chart because as of now it stays the same
7mn7mn

**Comment by rauleffting** [comment]
Hi guys, I took this PR to finish. It still needs to be connected to backend data. So, you don't need to review it now. I'll let you know when it's done! Thank you!
